sdk/storage/ci.yml

```
@@ -39,6 +39,7 @@ stages:
       CtestRegex: no-run
       Coverage: disabled
       LiveTestCtestRegex: azure-storage
+      CoverageReportPath: sdk/*/*cov_xml.xml
```

Why is this string different? Can we keep it consistent like the others?
```suggestion
      CoverageReportPath: sdk/*/*/*cov_xml.xml
```

---

It depends on where the test exe file is.
Storage has just one exe file for all the tests they run (blobs, files, shares) and the test exe is created on the main directory: sdk/storage/testFile.exe

For Azure core, and other services, we create a test exe for each package and it is created inside the package like
sdk/core/test/test_exe

I have already asked storage team to split their exe into 3.  Eventually, when they do that, they can then remove this.
For now, the pipeline allows to set the regext to find the reports.

---

> I have already asked storage team to split their exe into 3.

Is there an issue?

---

noup, before creating the issue, we need to create the Testing guidance and define some rules for it.
Then we would use that to request Storage to follow that.
Since we currently don't have any guide, it is allowed for any service to create the tests as they like, I think